src/util/prometheus.js

```diff
@@ -437,6 +437,12 @@ class Prometheus {
         type: 'counter',
         labelNames: ['libraryVersionId', 'version', 'type', 'success'],
       },
+      {
+        name: 'shopify_no_cartToken',
+        help: 'shopify_no_cartToken',
+        type: 'counter',
+        labelNames: ['sourceType', 'status'],
```

Comment:
Please double check the labelset. 

I checked the code and found that timestamp is in the label. Is it required? It will blow up the cardinality.